bahmni-emr-api/src/test/resources/TestingApplicationContext.xml

```diff
@@ -18,6 +18,11 @@
         <property name="mappingJarLocations">
             <ref bean="mappingJarResources"/>
         </property>
+        <property name="packagesToScan">
```

Comment:
this might increase load time, no? can it be narrower, if only required for the module pkgs?

Reply:
Thats right @angshu but this is the suggestion from openmrs org to get the test pass. please check this for more info
https://talk.openmrs.org/t/error-report-evaluation-failing-for-xls-reports-with-core-2-4/33428/9

Reply:
this might be thee reason for duplicate bean error as well, during test exec?